commands/bug/bug_show.go

```diff
@@ -91,7 +91,9 @@ func runBugShow(env *execenv.Env, opts bugShowOptions, args []string) error {
 		case "title":
 			env.Out.Printf("%s\n", snap.Title)
 		default:
-			return fmt.Errorf("\nUnsupported field: %s\n", opts.fields)
+			env.Out.Println()
```

Comment:
trying out that command, it doesn't make that much sense to have a new line, I think we can remove it entirely.

Reply:
I removed one of the two new-lines ... I thought maybe that was to separate an error output from the field output that might be directly above it.

Reply:
Let's remove both.